pkg/ingester/transfer.go

```diff
@@ -79,6 +79,33 @@ func (i *Ingester) TransferChunks(stream logproto.Ingester_TransferChunksServer)
 		if fromIngesterID == "" {
 			fromIngesterID = chunkSet.FromIngesterId
 			level.Info(util.Logger).Log("msg", "processing TransferChunks request", "from_ingester", fromIngesterID)
+
+			// Ring gossiping: check if "from" ingester is in LEAVING state. It should be, but we may not see that yet.
+			// If ingester is not LEAVING yet, we don't accept this transfer, as claiming tokens would end up
+			// with this ingester owning no tokens.
+			// Hopefully the leaving ingester will retry again.
+			v, err := i.lifecycler.KVStore.Get(stream.Context(), ring.ConsulKey)
+			if err != nil {
+				util.Logger.Log("msg", "TransferChunks error", "err", err)
```

Comment:
It seems inconsistent to use `util.Logger.Log` here (and in the other places below), where we generally always use `level.Error` in other places. Can we turn these into error-level messages?

Reply:
You're correct. This is slightly older copy of code from Cortex, I updated it there but not in my Loki repo :( Thanks!